idd/Energy+.idd.in

```diff
@@ -99616,6 +99639,7 @@ Output:Meter,
        \note RunPeriod, Environment, and Annual are the same
        \key Hourly
        \key Daily
+       \key Detailed
```

Comment:
Detailed isn't relevant for meters, because they are produced at the zone timestep.  If detailed is legal in the code, then fine to add here, but if we add it here, then it should also be in the next three objects as well.